src/EditModel/event-program/create-data-entry-form/CreateDataEntryForm.component.js

```diff
@@ -91,9 +106,16 @@ class CreateDataEntryForm extends Component {
                         />
                     )}
 
+                    {/* Super hacky to use the number 2 here */ ''}
+                    {/* I just didn't see another way */ ''}
```

Comment:
Just an idea to make it less 'hardcoded' would be to map Basic, Section, and CustomForm to a number instead of just setting `sectionFormIndex = 1`. So you could do something like`this.state.curTab === formIndex.basic`.